src/serverenvironment.h

```diff
@@ -205,7 +205,7 @@ class ServerEnvironment final : public Environment
 	*/
 
 	void addActiveBlockModifier(ActiveBlockModifier *abm);
-	ActiveBlockModifier *getActiveBlockModifierNoEx(const std::string &name);
+	bool setActiveBlockModifier(ActiveBlockModifier *abm);
```

Comment:
"set" feels like an odd name for this operation. maybe "update" or "replace", and add a doc comment that states something like `/// Replaces an ABM by name`?